stories/Atom/Navigation/ProgressBarNavigation/ProgressBarNavigation.scss

```diff
@@ -5,24 +5,32 @@
 // * progress navigation start *//
 .progress-container {
   background: $color-gray-300;
-  height: 12px;
+  height: 0.75rem;
   width: 100%;
 
   @include devicebreak(medium) {
-    height: 16px;
+    height: 1rem;
+  }
+
+  &.small, &.small .progress-bar{
+    height: 0.625rem;
+    
+    @include devicebreak(medium) {
+      height: 0.75rem;
+    }
   }
 }
 
 .progress-bar {
   background: $color-yellow;
```

Comment:
@Shamimkhan321 Can we add background color based on modifier?

Reply:
there is no specify for background color change.